common-pipeline/operators/uihints-v3-schema.json

```diff
@@ -662,6 +669,70 @@
           }
         }
       ]
+    },
+    "button_definition": {
+      "description": "Button to be displayed after the table's search bar",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Button identifier used to idenify the button in the callback function",
+          "type": "string"
+        },
+        "label": {
+          "description": "Button label to display. If an icon is specified as well, the icon will be shown to the right of the label.",
+          "type": "object",
+          "$ref": "#/definitions/resource_definition"
+        },
+        "description": {
+          "description": "Tooltip text to display when the button is hovered",
+          "type": "object",
+          "$ref": "#/definitions/resource_definition"
+        },
+        "icon": {
+          "description": "URL to .svg image to display",
+          "type": "string"
+        },
+        "carbon_icon": {
+          "description": "Host provided name of Carbon icon to display. A callback function is required for the host application to return the jsx icon object imported from @carbon/icons-react library",
+          "type": "object"
+        },
+        "enable": {
```

Comment:
Should this be enabled instead?

Reply:
"enabled" does sound better, but I was trying to keep consistent with what the toolbar config object expects https://github.com/elyra-ai/canvas/wiki/2.1-Config-Objects#toolbar-config-object

Should I go ahead and change it to "enabled"?

Reply:
I think so since users don't really know about the toolbar-config in common-properties.